Parse/Internal/Push/State/PFPushState.m

```diff
@@ -45,6 +45,19 @@ + (instancetype)stateWithState:(PFPushState *)state {
 #pragma mark - NSCopying
 ///--------------------------------------
 
+- (void)setPushDate:(NSDate *)pushDate {
+    if (self.pushDate != pushDate) {
+        NSTimeInterval interval = [pushDate timeIntervalSinceNow];
+        PFParameterAssert(interval > 0, @"Can't set the scheduled push time in the past.");
+        PFParameterAssert(interval <= 60 * 24 * 14, @"Can't set the schedule push time more than two weeks from now.");
```

Comment:
Change assertion message to 'fortnight from now'? (always wanted to have an excuse to use 'fortnight' :D)